root/static/scripts/edit/URLCleanup.js

```diff
@@ -748,6 +752,40 @@ const CLEANUPS = {
       return true;
     },
   },
+  instagram: {
+    match: [new RegExp("^(https?://)?([^/]+\\.)?instagram\\.com/", "i")],
+    type: LINK_TYPES.socialnetwork,
+    clean: function (url) {
+      var m = url.match(/^(https?:\/\/)?((?:[^\/]+\.)?instagram\.com)\/([^?#]+)(?:[?#].*)?$/);
+      if (m) {
+        var scheme = m[1] || 'https://';
```

Comment:
Added in https://github.com/metabrainz/musicbrainz-server/pull/441/commits/56b90d72ccc4a4654708bcbf1fd703128b468ded and https://github.com/metabrainz/musicbrainz-server/pull/441/commits/212e60c220c5270045f9dfe0d4ee2229927de11b (to be squashed after review).